server/services/sqs-messaging/sqs-consumer-factory.js

```diff
@@ -0,0 +1,28 @@
+const { Consumer } = require('sqs-consumer')
+const AWS = require('aws-sdk')
+
+class SqsConsumerFactory {
+  static configureSQS (config) {
+    return (config && config.accessKeyId && config.secretAccessKey)
```

Comment:
Could the SqsConsumerFactory be simplified if some config is pushed down into the queue config?

The Joi schema validation would prevent the accessKeyId and secretAccessKey from being undefined, and the region and waitTime could be pushed down to the queue configuration, settable via env vars.